src/main/java/net/clgd/ccemux/emulation/CCEmuX.java

```diff
@@ -85,9 +85,10 @@ public EmulatedComputer createComputer() {
 	 *
 	 * @param builderMutator
 	 *            Will be called after plugin hooks with the builder
+	 * @param turnOn
 	 * @return The new computer
 	 */
-	public EmulatedComputer createComputer(Consumer<EmulatedComputer.Builder> builderMutator) {
+	public EmulatedComputer createComputer(Consumer<EmulatedComputer.Builder> builderMutator, boolean turnOn) {
```

Comment:
Instead of having a `turnOn` argument, I think we should add a `startOn` field & method to `EmulatedComputer.Builder` class. If we default this to true we can keep the previous behaviour and don't have to add additional parameters.